client/src/pages/Profile.js

```diff
@@ -4,12 +4,14 @@ import ProfileDetailPanel from '../components/ProfileDetailPanel';
 import { withStyles, Button, Grid, Typography } from '@material-ui/core';
 import { getUser } from '../api/users';
 
+import './profile.css'
```

Comment:
We should probably stick with the css-in-js approach instead of importing plain css